ee/localserver/krypto-ec-middleware.go

```diff
@@ -380,6 +397,31 @@ func extractChallenge(r *http.Request) (*challenge.OuterChallenge, error) {
 	return challenge.UnmarshalChallenge(decoded)
 }
 
+func (e *kryptoEcMiddleware) responseWithoutHardwareSig(o *challenge.OuterChallenge, data []byte) ([]byte, error) {
+	msg := secureenclavesigner.SignResponseInner{
+		Nonce:     ulid.New(),
+		Timestamp: time.Now().UTC().Unix(),
+		// add the kolide:%s:kolide tags since the secure enclave cmd
+		// wont be execed and add them
+		Data: []byte(fmt.Sprintf("kolide:%s:kolide", string(data))),
```

Comment:
Optional -- maybe could add a small helper function in `secureenclavesigner` to do the `kolide:%s:kolide` formatting? Then `cmd/launcher/secure_enclave_darwin.go` could call that when it needs it too (also line 41 in `krypto-ec-middleware-response_other.go`), and we'd only have one place to remember/update it.